covid19-etl/etl/jhu_to_s3.py

```diff
@@ -163,12 +168,36 @@
         },
         ...
     }
+
+    (5) Choropleth IL map JSON by date:
+    Values are truncated or excluded to reduce file size
+    Date only added if C or D has changed from previous date
```

Comment:
isn't this going to slow things down on the frontend?
if you want to display the map for date Z, and the value is missing, you have to fetch the file for date Y, if it's still missing, you have to fetch date X...

Reply:
i am removing as it makes things more complicated for the went loop dev. it dosent slow things down that much as you just need a loop to run for each day in your dataset and add the skipped values, we are only dealing with hundreds of days so its not that big of a set.
I did this because im used to sites that get miliones of views a day and every Bit saved counts in that scenario